packages/core/src/Select/SelectItem.vue

```diff
@@ -58,9 +66,20 @@ const isFocused = ref(false)
 const textValue = ref(props.textValue ?? '')
 const textId = useId(undefined, 'reka-select-item-text')
 
-async function handleSelect(ev?: PointerEvent) {
+const SELECT_SELECT = 'select.select'
+
+async function handleSelectCustomEvent(ev: PointerEvent | KeyboardEvent) {
+  if (ev.defaultPrevented)
+    return
```

Comment:
I think we shouldn't have this conditional here, it will prevent the `select` from being emitted.

Reply:
It is needed because this logic is handling the initial mousedown event, when the dropdown just shows up. Without this check the dropdown just shows up on mousedown and disappears on mouseup.

So basically the check that we had previously - has moved from `handleSelect` to `handleSelectCustomEvent`, as this is the place now where we handle the original event. And the check that we have now in `handleSelect` is for actually preventing the artificially created event.